src/SFML/Graphics/RenderTexture.cpp

```diff
@@ -45,6 +45,14 @@ RenderTexture::RenderTexture() = default;
 RenderTexture::~RenderTexture() = default;
 
 
+////////////////////////////////////////////////////////////
+RenderTexture::RenderTexture(RenderTexture&& other) noexcept = default;
```

Comment:
```suggestion
RenderTexture::RenderTexture(RenderTexture&&) noexcept = default;
```

You can omit the parameter name for these defaulted functions.